main.go

```diff
@@ -309,6 +394,41 @@ func Exists(path string) bool {
 	return false
 }
 
+/*
+*	This function expands given input globs into matching paths on the server.
+ */
+func ExpandGlobs(cmdExec cmd_exec.CmdExec, paths []string, instance string) []string {
+	var newPaths []string
+	// iterate over each input path
+	for _, v := range paths {
+		// check if path is a glob
+		if strings.ContainsAny(v, "*?[]") {
+			dir := filepath.Dir(v)
+			out, _ := cmdExec.GetFile(appName, dir, instance)
+			body := strings.Split(string(out), "\n")
+			body = body[3 : len(body)-2]
+
+			//iterate over glob's directory
+			for _, w := range body {
+				cur := strings.SplitN(w, " ", 2)[0]
+				match, err := filepath.Match(filepath.Base(v), strings.TrimSuffix(cur, "/"))
+				check(err, "")
+				// append path to return list if glob pattern matches
+				if match && !strings.HasPrefix(cur, ".") {
```

Comment:
Yes, because the `dloader` ignores them as well when downloading directories or the entire application.